ovirtlago/testlib.py

```diff
@@ -209,7 +209,7 @@ def _instance_of_any(obj, cls_list):
     return any(True for cls in cls_list if isinstance(obj, cls))
 
 
-def assert_equals_within(func, value, timeout, allowed_exceptions=None):
+def assert_equals_within(func, value, timeout, allowed_exceptions=None, initial_wait=10):
```

Comment:
I also think that the initial wait can be reduced.

Not required for this patch:

1. We can also print to the log the name of the function and amount of time the test had to wait for a result, and 
    then do some fine-grained customization to the initial wait (per call) in the tests.

2. We can also consider using some kind of exponential backoff as a value for the sleep function.